Rocket.Chat/cs.lproj/Localizable.strings

```diff
@@ -141,3 +153,9 @@
 "chat.messages.list.empty" = "Nebyly nalezeny žádné zprávy";
 "chat.messages.pinned.list.title" = "Připnuté zprávy";
 "chat.messages.starred.list.title" = "Zprávy s hvězdičkou";
+
+// Settings
+"myaccount.settings.title" = "Settings";
+"myaccount.settings.contactus" = "Contact us";
+"myaccount.settings.license" = "License";
+"myaccount.settings.version" = "Version: %@ (%@)";
```

Comment:
@flashbackcz I know it sucks, but could you help us translating this new strings? We're working to integrate with LingoHub soon.

Reply:
Tomorrow I will do it (;

I have to admit that LingoHub is cool, there I translated some strings to czech too.